app/mailers/patient_mailer.rb

```diff
@@ -185,16 +185,46 @@ def assessment_email(patient)
   end
 
   def closed_email(patient)
-    return if patient&.email.blank?
+    if patient&.email.blank?
+      add_fail_history_closed_blank_field(patient, 'email')
+      return
+    end
 
-    @patient = patient
     @lang = patient.select_language
+    @contents = I18n.t(
+      'assessments.sms.closed.thank-you',
+      initials_age: patient&.initials_age('-'),
+      completed_date: patient.closed_at&.strftime('%m-%d-%Y'),
+      locale: @lang
+    )
     mail(to: patient.email&.strip, subject: I18n.t('assessments.email.closed.subject', locale: @lang)) do |format|
       format.html { render layout: 'main_mailer' }
     end
     History.monitoring_complete_message_sent(patient: patient)
   end
 
+  def closed_sms(patient)
+    if patient&.primary_telephone.blank?
+      add_fail_history_closed_blank_field(patient, 'primary phone number')
+      return
+    end
+    if patient.blocked_sms
+      TwilioSender.handle_twilio_error_codes(patient, TwilioSender::TWILIO_ERROR_CODES[:blocked_number][:code])
+      return
+    end
+
+    lang = patient.select_language
+    contents = I18n.t(
+      'assessments.sms.closed.thank-you',
+      initials_age: patient&.initials_age('-'),
+      completed_date: patient.closed_at&.strftime('%m-%d-%Y'),
+      locale: lang
+    )
+
+    TwilioSender.send_sms(patient, [contents])
```

Comment:
Single SMS has been changed relatively recently and you'll have to change/add this

`    threshold_hash = patient.jurisdiction.jurisdiction_path_threshold_hash
    message = { prompt: contents, patient_submission_token: patient.submission_token,
      threshold_hash: threshold_hash }

    TwilioSender.send_sms(patient, [message])`

Reply:
This should be resolved now